src/components/FilterPanel/FilterPanel.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { describe, it, expect, vi } from 'vitest';
+
+import { FilterPanel } from './FilterPanel';
+
+describe('FilterPanel', () => {
+  it('resalta el filtro seleccionado', () => {
+    render(
+      <FilterPanel
+        filters={[
+          { label: 'All Tests', value: 'all' },
+          { label: 'Upcoming', value: 'upcoming' },
+        ]}
+        selectedValue="all"
+        onSelect={() => {}}
+      />
+    );
+
+    const selectedButton = screen.getByTestId('filter-all');
```

Comment:
The test is using getByTestId('filter-all') to find the selected button. The project's testing guidelines recommend using semantic selectors (getByRole, getByLabelText) where possible instead of test ids.

**Sugerencia:**
```typescript
Use accessible queries. Example:

const selectedButton = screen.getByRole('radio', { name: /All Tests/i });
expect(selectedButton).toHaveAttribute('aria-checked', 'true');
// or use toBeChecked() matcher if available
```